api/src/accounts.rs

```diff
@@ -62,12 +68,11 @@ impl AccountsApi {
 
     /// Get account resources
     ///
-    /// This endpoint returns all account resources at a given address at a
-    /// specific ledger version (AKA transaction version). If the ledger
-    /// version is not specified in the request, the latest ledger version is used.
+    /// Retrieves all account resources for a given account and a specific ledger version.  If the
```

Comment:
I really prefer the way it was, with `at a specific ledger version`. The other change is fine. So:

> Retrieves all account resources for a given account at a specific ledger version.